disco_aws_automation/disco_vpc.py

```diff
@@ -159,6 +173,46 @@ def networks(self):
         }
         return self._networks
 
+    def _create_new_meta_networks(self):
```

Comment:
After reading more, looks like you only want the network @property method to load the metanetworks, not create them, therefore no need to calculate all the metanetwork cidrs.